pom.xml

```diff
@@ -2,12 +2,14 @@
     <modelVersion>4.0.0</modelVersion>
 
     <groupId>org.spectrumauctions</groupId>
-    <artifactId>sats-core</artifactId>
```

Comment:
This looks dangerous; how are people notified the have to change the artifactId when checking for updates. 

However, I don't have a better proposition (keeping sats-core doesn't look to nice to me). We'll have to send out a notification to our subscribers and known users.

Also, somwhere, we have to declare how sats-core, sats-opt and sats-clt versions map to the new version, to keep the possibility of reproducing results for our users nice and easy :-)

Reply:
I agree that we will have to notify the users about the change, but I think it's the cleanest way like this: Since the version is now bumbed to 0.6.0, all versions below 0.6.0 will remain split (the artifacts are not lost on maven central), and from 0.6.0. on upwards, there's only one `sats`.

This would also solve the reproducibility challenge, right? If someone wants to reproduce a clt-0.5.1 output, he/she just declares this dependency, and it will pull the according core dependency. Or did I misunderstand your point?